crates/polars-core/src/utils/mod.rs

```diff
@@ -34,6 +35,35 @@ impl<T> Deref for Wrap<T> {
     }
 }
 
+// This is giant trust me bro
+#[repr(transparent)]
+#[derive(Debug)]
+pub struct RegexWrap<T>(pub T);
+
+impl<T> AsRef<T> for RegexWrap<T> {
+    fn as_ref(&self) -> &T {
+        &self.0
+    }
+}
+
+impl<T> Default for RegexWrap<T> {
```

Comment:
This may be problematic? I feel it will be, not sure how though

Reply:
Done